src/common/components/arbeidsoppgaver/CustomAlertWithoutIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import React, {ReactNode} from "react";
+
+interface Props {
+    children: ReactNode;
+}
+// Should not be used. Should be replaced by designsystem alert
+export const CustomAlertWithoutIcon = ({ children }: Props) => {
```

Comment:
Kanskje bare bytte med designsystem? har ikke støtte for de classname-tingene uansett

Reply:
Designsystemet sin komponent Alert inneholder et ikon, som allerede vises på statusen til arbeidsoppgaven. Derfor ville ikke Espen ha ikonet to steder. Det går ikke an å skjule ikonet i Alert-komponenten, derfor valgte jeg å hacke til en variant som bruker de samme klassene som designsystemet. Har du noen bedre forslag?

Og hva mener du med at vi ikke har støtte for classname-tingene?

Reply:
Ah. Da skjønner jeg!

Oi, så ikke så tydelig hvilke classnames det var du hadde lagt inn. Trodde det var gamle .less-ting (har ikke lagt inn støtte for less). Da trekker jeg tilbake kommentaren min!